index.html

```diff
@@ -65,6 +65,67 @@ <h3>Use case #2: Efficiency enhancements during RTC</h3>
         </p>
       </section>
     </section>
+    <section id="examples">
+      <h2>Examples</h2>
+      <p>The <a data-cite="screen-capture/#example">basic example from
+        the Screen Capture specification</a> is extended below by passing
+        to <code>getDisplayMedia()</code> a <code>CaptureController</code>
+        object. An event handler is set on that object in order to receive
+        {{CapturedMouseEvent}}s with the mouse coordinates over the captured
+        surface.
+      </p>
+<pre class="example highlight">try {
+  let controller = new CaptureController();
+  controller.oncapturedmousechange = (event) => {
+    console.log(`Mouse coordinates: x=${event.surfaceX}, y=${event.surfaceY}`);
+  };
+  let mediaStream = await navigator.mediaDevices.getDisplayMedia({
+    video: true,
+    controller: controller,
+  });
```

Comment:
Since `video`value  is `true` by default, you could rewrite it to 
```js
const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
```

Reply:
Yes, as mentioned above I was copying an existing example so I thought it was clearer to not modifying the line. but that's no longer a concern now.